### PR TITLE
radware,slack,sophos,squid,suricata,symantec,ti_*,tomcat: remove duplicate fields

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

Remove duplicate field definitions.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- For #4398

### CODE REVIEW

File: packages/symantec_endpoint/data_stream/log/fields/agent.yml [hunk 1]
```diff
@@ -63,10 +63,7 @@
   type: group
   fields:
     - name: id
-      level: core
-      type: keyword
-      ignore_above: 1024
-      description: Unique container id.
+      external: ecs
```
Comment (kcreddy) [inline packages/symantec_endpoint/data_stream/log/fields/agent.yml:66]: Should be just removing the `id` field here, instead of adding `external:ecs`

**Review by kcreddy — APPROVED** [review]
Minor change requested. Rest LGTM 👍🏼

File: packages/symantec_endpoint/data_stream/log/fields/agent.yml (hunk 1 above)
Comment (efd6) [reply packages/symantec_endpoint/data_stream/log/fields/agent.yml:66]: No, I've left the `container.id` field in this file and removed the `container.id` field definition in ecs.yml.

Comment (kcreddy) [reply packages/symantec_endpoint/data_stream/log/fields/agent.yml:66]: Got it, looks good to me Dan 👍🏼

**Pull request merged**